pkg/vault/contracts/Vault.sol

```diff
@@ -1031,39 +1018,40 @@ contract Vault is IVaultMain, VaultCommon, Proxy {
      * Splitting the fees and event emission occur during fee collection.
      * Should only be called in a non-reentrant context.
      *
-     * @return totalFeesRaw Sum of protocol and pool creator fees raw
+     * @return totalSwapFeeAmountRaw Total swap fees raw (LP + aggregate protocol fees)
+     * @return aggregateSwapFeeAmountRaw Sum of protocol and pool creator fees raw
      */
     function _computeAndChargeAggregateSwapFees(
```

Comment:
I'd say the name of this function should change, since it's computing the total and aggregate swap fees. But, I understand that it only charge the aggregate swap fees, so probably it's better to leave it as is.

Reply:
Could just be `_computeAndChargeSwapFees`, though I think it's also fine as is.

There is a comment in the unchanged code saying we're not emitting events, though, which is now a bit misleading.

"Note that this computes the aggregate total of the protocol fees and stores it, without emitting any events."

Maybe just add "The total swap fees are reported in the `PoolBalancesChanged` event"